tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -46,18 +46,6 @@ const deleteUpgradeLogs = async () => {
   await utils.logsDb.bulkDocs(logs);
 };
 
-// ToDo: Remove once 4.4 is released. Because it needs selectors targeting the previous cht version.
-const oldLogout = async () => {
-  await commonPage.openHamburgerMenu();
-  await (await commonPage.logoutButton()).waitForClickable();
-  await (await commonPage.logoutButton()).click();
-  $('.modal-dialog .modal-body').waitForDisplayed();
-  const submitBtn = $('.modal-dialog a.btn.submit');
-  await (await submitBtn).waitForClickable();
-  await (await submitBtn).click();
-  await browser.pause(100); // Wait for login page js to execute
-};
-
```

Comment:
The function below this has a comment that it should be removed too though I'm not sure what to replace it with.